interview-questions.md

```diff
@@ -6,54 +6,54 @@ First, without external resources. Challenge yourself to answer from memory as i
 
 Then, research the question to expand on your answer. Even if you feel you have answered the question completely, there is always something more to learn. Write your researched answer in your OWN WORDS.
 
-Type your initials here to confirm you read the instructions:
+Type your initials here to confirm you read the instructions: KM
 
 INSTRUCTOR EXAMPLE: What is a conditional statement?
 
-Your answer:
+Your answer: if/else statements
 
-Researched answer:
+Researched answer: Conditional statements control behavior in JavaScript and determine whether or not pieces of code can run determined by if it is true or false.
 
 1. What is git? What is the difference between git and Github?
 
-Your answer:
+Your answer: Git is the pipeline between your local computer and github to upload files, codes, and projects. Github is the cloud that stores the code, projects, and files.
 
-Researched answer:
+Researched answer: Git is a version control system that lets you manage and keep track of your source code history. GitHub is a cloud-based hosting service that lets you manage Git repositories.
 
 2. Which JavaScript operators will return a Boolean value?
 
-Your answer:
+Your answer: >, <, =, ==, ===
 
-Researched answer:
+Researched answer: NOT (!)(!!), AND (&&), & OR (||) operators
 
-1. What is the difference between an array's index and its value?
+3. What is the difference between an array's index and its value?
 
-Your answer:
+Your answer: what is stated within an array are the values itself. Indexing is outputing the location of the values within the array.
 
-Researched answer:
+Researched answer: Indexing is an operation that pulls out a select set of values from an array. The index of a value in an array is that value's location within the array. 
```

Comment:
The question does not refer to indexing, but is asking the difference between an index and a value in an array.